ihmc-perception/src/test/java/us/ihmc/perception/cuda/examples/ExampleCUDAKernel2D.java

```diff
@@ -0,0 +1,124 @@
+package us.ihmc.perception.cuda.examples;
+
+import org.bytedeco.cuda.cudart.CUstream_st;
+import org.bytedeco.cuda.cudart.dim3;
+import org.bytedeco.cuda.global.cudart;
+import org.bytedeco.opencv.global.opencv_core;
+import org.bytedeco.opencv.opencv_core.GpuMat;
+import org.bytedeco.opencv.opencv_core.Mat;
+import org.bytedeco.opencv.opencv_core.Scalar;
+import us.ihmc.perception.cuda.CUDAKernel;
+import us.ihmc.perception.cuda.CUDAProgram;
+import us.ihmc.perception.cuda.CUDAStreamManager;
+
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.util.Objects;
+
+/**
+ * This is an example of using a more complex CUDA kernel that takes in a JavaCV {@link Mat} object. We do this often for kernels, so this example walks you
```

Comment:
```suggestion
 * This is an example of using a more complex CUDA kernel that takes in an OpenCV {@link Mat} object. We do this often for kernels, so this example walks you
```

I'd say `Mat` is an OpenCV object that we use through JavaCPP, and JavaCV happens to use it as well. Although I kinda see an argument for saying JavaCV. So this is just an idea, not necessarily a request to make the change